### DISCUSSION TITLE
How can I disable paragraph when using this tool?

### DISCUSSION THREAD

**Discussion by AlexPD93** [hata6502/editorjs-layout#42] [Q&A]
I am trying to use this layout tool in a custom project. However, I would like to disable the use of the default paragraph tool that comes with editor js as I am making my own custom plugins. Could you explain how I can do this? Thank you

I have tried: 

`
const editorJSConfig = {
  tools: {
    paragraph: false,
}
`

But this doesn't work.

**Comment by hata6502** [comment]
https://editorjs.io/configuration/#change-the-default-block

Before using editorjs-layout, first find a way to disable the paragraph tool in Editor.js.

**Reply by AlexPD93** [reply]
Thank you for your reply. 

I have changed the default block in the layout tool to a custom tool and disabled the paragraph tool in the editor js instance but it still doesn't work
